docs/faq.md

```diff
@@ -6,6 +6,49 @@
 
 See [Guide: Struct](./guide/struct.md)
 
+## Skip conversion (aka shallow copy), e.g for time.Time
+
+Many structs like `time.Time` or `net/url.URL` are best to
+just copy instead of convert field-by-field.
```

Comment:
I'd rather improve the existing unexported-field docs, as I'd disagree with this statement. Basically, one of the three options is the right choice in case of an error like below. There is no one fits all solution. Sure for time.Time it's shallow copy, but the text is currently generic "many structs like".

Reply:
Ye that's fine. I don't know what you have in mind so I'm leaving that up to you.

I'm closing this PR then

Reply:
If time.Time is supported natively then it's likely not a big problem anymore that needs a faq entry.